openupgrade_scripts/scripts/mail_group/15.0.1.0/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,107 @@
+---Models in module 'mail_group'---
+new model mail.group
+new model mail.group.member
+new model mail.group.message
+new model mail.group.message.reject [transient]
+new model mail.group.moderation (renamed from mail.moderation in module mail)
+---Fields in module 'mail_group'---
+mail_group   / mail.group               / _inherits                     : NEW _inherits: {'mail.alias': 'alias_id'}
+mail_group   / mail.group               / access_group_id (many2one)    : NEW relation: res.groups, hasdefault
+mail_group   / mail.group               / access_mode (selection)       : NEW required, selection_keys: ['groups', 'members', 'public'], req_default: function, hasdefault
+mail_group   / mail.group               / active (boolean)              : NEW hasdefault
+mail_group   / mail.group               / alias_id (many2one)           : NEW relation: mail.alias, required
+mail_group   / mail.group               / description (text)            : NEW
+mail_group   / mail.group               / image_128 (binary)            : NEW attachment: True
+mail_group   / mail.group               / mail_group_message_ids (one2many): NEW relation: mail.group.message
+mail_group   / mail.group               / member_ids (one2many)         : NEW relation: mail.group.member
+mail_group   / mail.group               / moderation (boolean)          : NEW
+mail_group   / mail.group               / moderation_guidelines (boolean): NEW
+mail_group   / mail.group               / moderation_guidelines_msg (html): NEW
+mail_group   / mail.group               / moderation_notify (boolean)   : NEW
+mail_group   / mail.group               / moderation_notify_msg (html)  : NEW
+mail_group   / mail.group               / moderation_rule_ids (one2many): NEW relation: mail.group.moderation
+mail_group   / mail.group               / moderator_ids (many2many)     : NEW relation: res.users
+mail_group   / mail.group               / name (char)                   : NEW required
+# DONE (in mail): pre-migration: filled with mail.channel records that had email_send = True
```

Comment:
I would put instead `# NOTHING TO DO: Done in mail pre-migration`, but isn't there lines for this in the `mail` work file?